services/src/main/java/org/jboss/windup/web/services/rest/WindupEndpointImpl.java

```diff
@@ -55,18 +65,49 @@
     private Queue executorQueue;
 
     /**
+     * To avoid problems with passing detached entities,
+     * load related entities during cloning
+     */
+    protected AnalysisContext cloneExecution(AnalysisContext originalContext)
+    {
+        Set<Package> allPackages = new HashSet<>();
+
+        allPackages.addAll(originalContext.getIncludePackages());
+        allPackages.addAll(originalContext.getExcludePackages());
+
+        List<Long> packageIds = allPackages.stream().map(aPackage -> aPackage.getId())
+                .collect(Collectors.toList());
+
+        AnalysisContext clone = originalContext.clone();
```

Comment:
Yes, it would. Although I'm getting a little confused about the use case now. If the data in the request doesn't match the data in the database, it seems that it wouldn't be possible to reload them from the persistence context either? It seems like this might be indicating that the Analysis Context being deserialized isn't really complete?